packages/twenty-front/src/modules/settings/data-model/object-details/components/SettingsObjectFieldItemTableRow.tsx

```diff
@@ -17,11 +18,12 @@ import { SettingsObjectFieldDataType } from './SettingsObjectFieldDataType';
 
 type SettingsObjectFieldItemTableRowProps = {
   ActionIcon: ReactNode;
-  fieldMetadataItem: FieldMetadataItem;
+  fieldMetadataItem: FieldMetadataItem & { fieldType: string | boolean };
```

Comment:
fieldType is redundant, we already have a type property on fieldMetadataItem which should be used.

Reply:
Removed the redundant field